go-schema-gen/generate.go

```diff
@@ -16,6 +17,32 @@ func GenerateSchema(title string, configObject interface{}) *jsonschema.Schema {
 		DoNotReference: true,
 	}
 	var schema = reflector.ReflectFromType(reflect.TypeOf(configObject))
+	schema.AdditionalProperties = json.RawMessage("true")
```

Comment:
uber nit: This could probably also be `nil` to reduce some noise in the schemas.

Reply:
Good point! Done and I'll get this merged after the checks finish